extension/src/popup/components/signTransaction/Operations.tsx

```diff
@@ -421,6 +470,45 @@ export const Operations = ({
                   operationValue={highThreshold}
                 />
               ) : null}
+              {sponsoredId ? (
+                <KeyValueWithPublicKey
+                  operationKey="Sponsored Id"
+                  operationValue={sponsoredId}
+                />
+              ) : null}
+              {account ? (
+                <KeyValueWithPublicKey
+                  operationKey="Account"
+                  operationValue={account}
+                />
+              ) : null}
+              {source ? (
+                <KeyValueWithPublicKey
+                  operationKey="Source"
+                  operationValue={source}
+                />
+              ) : null}
+              {claimants && claimants.length
+                ? claimants.map(({ _destination }, index) => (
+                    /* eslint-disable react/no-array-index-key */
+                    <div key={`${_destination}${index}`}>
+                      {/* eslint-enable */}
+                      <SubHeaderEl>Claimant {index + 1}</SubHeaderEl>
+                      <KeyValueWithPublicKey
+                        operationKey="Destination"
+                        operationValue={_destination}
+                      />
+                      {/* TODO: Add appicable predicate UI */}
+                    </div>
+                  ))
+                : null}
+              {Object.entries(rest).map(([k, v]) =>
+                React.isValidElement(v) ? (
+                  <div key={k}>
+                    <KeyValueList key={k} operationKey={k} operationValue={v} />
+                  </div>
+                ) : null,
+              )}
```

Comment:
This bit will simply show a key/value for any operations we've missed in `OPERATION_TYPES`